Dockerfile

```diff
@@ -1,30 +1,14 @@
-FROM runtimeverificationinc/ubuntu:bionic
+ARG K_COMMIT
+FROM runtimeverificationinc/kframework-k:ubuntu-bionic-${K_COMMIT}
 
-RUN    apt-get update                                                        \
-    && apt-get upgrade --yes                                                 \
-    && apt-get install --yes                                                 \
-        autoconf bison clang-8 cmake curl flex gcc libboost-test-dev         \
-        libcrypto++-dev libffi-dev libjemalloc-dev libmpfr-dev libprocps-dev \
-        libprotobuf-dev libsecp256k1-dev libssl-dev libtool libyaml-dev      \
-        libz3-dev lld-8 llvm-8-tools make maven netcat-openbsd opam          \
-        openjdk-11-jdk pandoc pkg-config protobuf-compiler python3           \
-        python-pygments python-recommonmark python-sphinx rapidjson-dev time \
-        z3 zlib1g-dev
+RUN    apt-get update         \
+    && apt-get upgrade --yes  \
```

Comment:
Why do run updates and upgrades? Wouldn't the K container have the right versions of it's dependencies?